gensim/downloader.py

```diff
@@ -197,9 +201,15 @@ def info(name=None):
             return information['models'][name]
         else:
             raise ValueError("Incorrect model/corpus name")
-    else:
+
+    if not show_only_latest:
         return information
 
+    return {
+        "corpora": {name: data for (name, data) in information['corpora'].items() if data.get("latest", True)},
```

Comment:
Should the default be `True`?

Reply:
Yes, It is much easier when the new version is loaded set `latest: False` for the old version, instead of set `latest` flag to all data as the required field.